source/pasta/carbonara.rst

```diff
@@ -0,0 +1,32 @@
+Carbonara
+=========
+
+Details
+-------
+
+* Number of servings: 4 per 1 lbs box of spaghetti
+* Cook time: 30 Minutes
+
+Ingredients
+-----------
+
+* Spaghetti
+* 1 egg per serving
+* 1 lbs of bacon
```

Comment:
In the other recipes, we don't use "of". We should stay consistent with formatting.

Reply:
Also ingredient capitalizations, I'm okay with either capital or not, but we need to keep it uniform.